src/main/java/io/clue2solve/aws/bedrock/springboot/starter/config/LlamaProperties.java

```diff
@@ -9,12 +9,26 @@
 import java.util.List;
 import java.util.Optional;
 
+/**
+ * Sample properties aws.bedrock.model.llama2.modelId=meta.llama2-13b-chat-v1
+ * aws.bedrock.model.llama2.topP=0.9 aws.bedrock.model.llama2.temperature=0.5
```

Comment:
Careful. Line break. Each key-value pair should appear on separate line for readability.